letsencrypt-apache/letsencrypt_apache/configurator.py

```diff
@@ -1,4 +1,6 @@
 """Apache Configuration based off of Augeas Configurator."""
+# pylint: disable=too-many-lines
```

Comment:
Have you thought about moving standalone functions (all below and including `apache_restart`) to a separate file?